pkg/providers/snow/creds.go

```diff
@@ -2,27 +2,27 @@ package snow
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/aws/eks-anywhere/pkg/aws"
 )
 
-type bootstrapCreds struct {
-	snowCredsB64 string
-	snowCertsB64 string
-}
-
 func (p *SnowProvider) setupBootstrapCreds() error {
 	creds, err := aws.EncodeFileFromEnv(eksaSnowCredentialsFileKey)
 	if err != nil {
 		return fmt.Errorf("failed to set up snow credentials: %v", err)
 	}
-	p.bootstrapCreds.snowCredsB64 = creds
+	if err := os.Setenv(snowCredentialsKey, creds); err != nil {
```

Comment:
we deliberately avoided setting the envs in the code and instead, used a struct to pass the secrets around so it can be used in different cases (both cli and controllers)